packages/node_modules/@node-red/nodes/locales/pt-BR/function/16-range.html

```diff
@@ -0,0 +1,44 @@
+<!--
+  Copyright JS Foundation and other contributors, http://js.foundation
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<script type="text/html" data-help-name="range">
+<p> Mapeia um valor numérico para um intervalo diferente. </p>
+    <h3> Entradas </h3>
+    <dl class = "message-properties">
+        <dt> carga útil <span class = "property-type"> número </span> </dt>
+        <dd> A carga útil <i> deve </i> ser um número. Qualquer outra coisa tentará ser
```

Comment:
Change to:
`        <dd> O payload <i> deve </i> ser um número. Qualquer outra coisa tentará ser`